packages/circuit-ui/components/ListItem/ListItem.tsx

```diff
@@ -401,40 +394,40 @@ export const ListItem = forwardRef(
               </DetailsContainer>
             )}
           </MainContainer>
-          {shouldRenderSuffixContainer && (
-            <SuffixContainer
-              hasLabel={!!suffixLabel}
+          {shouldRenderTrailingContainer && (
+            <TrailingContainer
+              hasLabel={!!trailingLabel}
               isNavigation={isNavigation}
             >
-              <SuffixChevronContainer>
-                {typeof suffixLabel === 'string' ? (
+              <TrailingChevronContainer>
+                {typeof trailingLabel === 'string' ? (
```

Comment:
Nittiest of nitpicks: you could refactor this (and the use cases below) to use the `isString` type check util. The util does the same check, but this would make it consistent with the `isFunction` check above 🤷‍♂️

Reply:
(I see you've adjusted one of the type checks, but not the ones further down the file)

Reply:
Thanks, done in "not such a nitty"